pkg/compact/downsample/downsample.go

```diff
@@ -695,6 +695,12 @@ func (it *AverageChunkIterator) Next() bool {
 	return true
 }
 
+func (it *AverageChunkIterator) Seek(t int64) bool {
+	for it.t < t && it.Next() {
```

Comment:
that's cheating =D

Reply:
But works :) would this cause any performance issues?

Reply:
We don't use it I think, so I would error out (`it.err = errors.New("seek used, but not implemented")`)